edge-modules/TestAnalyzer/TestOperationResult.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestAnalyzer
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class TestOperationResult
+    {
+        public TestOperationResult(
+            string source,
+            string resultType,
+            string result,
+            DateTime createdAt)
+        {
+            this.Source = Preconditions.CheckNonWhiteSpace(source, nameof(source));
+            this.ResultType = Preconditions.CheckNonWhiteSpace(result, nameof(resultType));
+            this.Result = Preconditions.CheckNonWhiteSpace(result, nameof(result));
+            this.CreatedAt = createdAt;
+        }
+
+        public string Source { get; }
+
+        public string ResultType { get; }
+
+        public string Result { get; }
```

Comment:
>Result [](start = 22, length = 6)

maybe call it ResultDetailsJson or ResultContentJson?

Reply:
in order to backward compatible, Result property can be just the value in plain text.